src/main/java/seedu/duke/commands/OpenLessonLinkCommand.java

```diff
@@ -49,8 +51,8 @@ public boolean isExit() {
      * Prints list of links opened.
      *
      * @param lessonList ArrayList of lessons in specified module.
-     * @param indices Indices of links to open.
-     * @param ui Instance of UI.
+     * @param indices    Indices of links to open.
+     * @param ui         Instance of UI.
      */
```

Comment:
Should we do it this way for the rest of the code?

Reply:
idm